sentry/src/main/java/io/sentry/ReplayController.java

```diff
@@ -0,0 +1,17 @@
+package io.sentry;
+
+import org.jetbrains.annotations.ApiStatus;
+import org.jetbrains.annotations.NotNull;
+
+@ApiStatus.Internal
+public interface ReplayController {
```

Comment:
I hate to say that, but shouldn't it be `IReplayController`? 😅

Reply:
myman 😅 I hate to say that, but you're right haha

Reply:
although, I see that sometimes we omit the `I` like in `EventProcessor` or `ObjectReader/Writer`, so maybe we can omit it here too? I don't expect us implementing replay for anything else besides Android tbh 😅

Reply:
Let's omit it then 👍